### ISSUE TITLE
Performance - omit 1.0000 when writing out LP file

### ISSUE THREAD

**Issue by willu47** [PyPSA/linopy#112]
linopy LP files are created with lots of extra characters when the coefficients are 1:

```
c2:
-1.000000 x6
+1.000000 x8
=
0.000000
```
These can be omitted:
```
c2:
- x6 + x8 = 0.00000
```

**Comment by willu47** [comment]
I've tested with Gurobi, GLPK and CBC and those seem fine.  I don't have access to other solvers at the moment.

**Comment by FabianHofmann** [comment]
closed by #114
